### ISSUE TITLE
spacemacs/report-issue fails poorly if I'm not logged into github

### ISSUE THREAD

**Issue by dwang20151005** [syl20bnr/spacemacs#5902]
#### Description
spacemacs/report-issue appears to assume that I'm running emacs on a machine whose default browser is logged into github.  This isn't true for me, which is fine.  But the failure mode is a generic github page that says "this page took way too long to load", which is hardly helpful.

If it's not possible to fail in a less misleading way, could we do away with having spacemacs/report-issue open the browser?  I'd be happy to copy-paste this template (with the system info filled in) out of a buffer into a github form that I've opened manually.

#### Reproduction guide
- Start Emacs on a box where your default browser isn't logged into github
- Call spacemacs/report-issue

*Observed behaviour:*
Browser opens a github error page that says, "This page took way too long to load."

*Expected behaviour:*
Browser opens with the github page for creating a new issue, or at least a github login page.

#### System Info
- OS: gnu/linux
- Emacs: 24.5.2
- Spacemacs: 0.105.13
- Spacemacs branch: chrome-layer-plain-text-gmail (rev. 107b53f)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-return-key-behavior 'complete auto-completion-tab-key-behavior 'cycle auto-completion-enable-help-tooltip t)
 (chrome :variables chrome-gmail-rich-text-format nil)
 emacs-lisp
 (jane :variables jane-jane-elisp-path "/usr/local/home/dwang/workspaces/jane/emacs/dev/spacemacs/jane-tweaks/rewrite/ share /app/emacs/" Omake\.Server\.program "/j/office/app/emacs/prod/jane-elisp/bin/build-manager")
 markdown org sql
 (version-control :variables version-control-diff-tool 'diff-hl)
 vinegar w3m)
```

#### Backtrace
```

```

**Comment by JAremko** [comment]
:bug:  :boom:  :gun:  @StreakyCobra

**Comment by syl20bnr** [comment]
Pew pew!!

**Comment by StreakyCobra** [comment]
The plan is to make `SPC h d I` open the issue in a new buffer in Markdown mode, allowing people to edit, and then to set a key binding to open a new browser tab with the issue pre-filled. This will also solve your problem. I just need to find time :-/

**Comment by syl20bnr** [comment]
That would be fantastic, a bit like editing a commit message.

**Comment by dwang20151005** [comment]
I think I know how to do this.  I can try to give you a pull request if you'd like.  Although my elisp is pretty bad, so feel free to tell me that the review would be more work than I'd save you.

**Comment by StreakyCobra** [comment]
Go ahead, I'll not be the only one that can review it :-)

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
